### ISSUE TITLE
Imgur embeds cause mail to go to spam

### ISSUE THREAD

**Issue by qaisjp** [HackAssistant/registration#235]
(apparently)

We should embed these as an attachment and use it, as per [this page](https://stackoverflow.com/questions/9110091/base64-encoded-images-in-email-signatures/9110164#9110164)

**Comment by casassg** [comment]
The other option is to host them statically in GitHub pages? We need to make sure that base64 get loaded correctly on all email clients.

**Comment by casassg** [comment]
Found this: https://djangosnippets.org/snippets/285/

**Comment by qaisjp** [comment]
The other option is hosting the static files on my.hackwhatever.tld but I still think attaching the image is safer. I think GitHub pages will still send it to spam because the sender domain != Image domain.

**Comment by faix** [comment]
What about hosting it in aws with S3 and using it as a cdn?

**Comment by casassg** [comment]
I feel the best option is going to host it on the same server as static files. May be difficult to debug on local, so maybe we can use imgur as a fallback when DEBUG mode is on. Keeping them on the project makes it more manageable and reduces the chances of getting into SPAM as it will be from the same domain.

**Comment by qaisjp** [comment]
that has the same issue of hosting images on a foreign domain.

**Comment by casassg** [comment]
Fair. Then we need to explore: https://djangosnippets.org/snippets/285/

**Comment by qaisjp** [comment]
Btw an easier option would just be to use this commit https://github.com/compsoc-edinburgh/hackassistant-registration/commit/c7449a071498b4b23698855c95680c2ec0aaafce — not sure if image support is worth all the effort

**Comment by casassg** [comment]
Problem with Base64 links is that Outlook blocks it. Will check locally. Interesting article about different alternatives:  https://sendgrid.com/blog/embedding-images-emails-facts/

**Comment by casassg** [comment]
Just tested. Images don't work correctly on GMail for Android/Web. Will try to do the attachment ones as suggested before.

**Comment by qaisjp** [comment]
Yup, I initially tried base64 and encountered the same issue with Gmail
